lib/internal/test_runner/reporter/tap.js

```diff
@@ -233,8 +233,8 @@ function jsToYaml(indent, name, value) {
     }
 
     if (errIsAssertion) {
-      result += jsToYaml(indent, 'expected', errExpected);
-      result += jsToYaml(indent, 'actual', errActual);
+      result += jsToYaml(indent, 'expected', errExpected, true);
+      result += jsToYaml(indent, 'actual', errActual, true);
```

Comment:
Should we `util.format` those, instead of stringify them?

Reply:
Do you mean `JSON.stringify`? why would that be better than `util.format`?